packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html

```diff
@@ -71,7 +71,7 @@
                <template for:each={values} for:item="v">
                 <c-quantic-facet-value 
                   onselectvalue={onSelectValue} 
-                  key={v.start} 
+                  key={v.index} 
```

Comment:
I wouldn't use the index as the key because, elements won't be updated unless the key changes. It would be safer to use the range as the key.